Update README.md expand template (#5)
viam-labs/generator-viam-module: generators/app/templates/python/README.md

# <%= name %> modular service

This module implements the [<%= api_namespace %> <%= api_name_lower %> API](https://github.com/<%= api_namespace %>/<%= api_name_lower %>-api) in a <%= model %> model.

> With this model, you can...

Author: @npentrel is it ok to use "..." like this in a template? Can reframe if not

## Requirements

> Add instructions here for any requirements.

Reviewer: We should use italics or brackets to indicate placeholder text that the module author should change:

```suggestion
_Add instructions here for any requirements._
```

Reviewer 2: While I agree this is an improvement, that's also not how it's currently used in this template, so we should change that throughout

``` bash
```

## Build and Run

To use this module, follow these instructions to [add a module from the Viam Registry](https://docs.viam.com/registry/configure/#add-a-modular-resource-from-the-viam-registry) and select the `<%= api_namespace %>:<%= api_name_lower %>:<%= model %>` model from the [`<%= model %>` module](https://app.viam.com/module/<%= api_namespace %>/<%= model %>).

## Configure your <%= api_name_lower %>

> [!NOTE]  

> > Before configuring your <%= api_name_lower %>, you must [create a robot](https://docs.viam.com/manage/fleet/robots/#add-a-new-robot).

Reviewer 3: Shouldn't we say machine now?

Author: Oh yes & probably should update the link, good catch

> > Before configuring your <%= api_name_lower %>, you must [create a robot](https://docs.viam.com/manage/fleet/robots/#add-a-new-robot).

Author: ```suggestion
> Before configuring your <%= api_name_lower %>, you must [create a machine](https://docs.viam.com/manage/fleet/machines/#add-a-new-machine).
```

Navigate to the **Config** tab of your robot’s page in [the Viam app](https://app.viam.com/).
Click on the **Components** subtab and click **Create component**.
Select the `<%= api_name_lower %>` type, then select the `<%= model %>` model. 
Enter a name for your <%= api_name_lower %> and click **Create**.

On the new component panel, copy and paste the following attribute template into your <%= api_name_lower %>’s **Attributes** box:

```json
{

>   <INSERT SAMPLE ATTRIBUTES>

Reviewer 2: I think the angular brackets are fine for placeholder values, since this is an entire blog I think this might look better?
```suggestion
  TODO: INSERT SAMPLE ATTRIBUTES
```

Author: Sounds good

}
```

> [!NOTE]  
> For more information, see [Configure a Robot](https://docs.viam.com/manage/configuration/).

### Attributes

The following attributes are available for `<%= api_namespace %>:<%= api_name_lower %>:<%= model %>` <%= api_name_lower %>s:

| Name | Type | Inclusion | Description |
| ---- | ---- | --------- | ----------- |

> | `` | string | **Required** |  |
> | `` | string | Optional |  |

Reviewer 2: Maybe? 
```suggestion
| `todo1` | string | **Required** |  TODO |
| `todo2` | string | Optional |  TODO |
```

### Example Configuration

> ```json

Reviewer 3: Is this now redundant?

Author: We've been having the attributes and the example configuration separately to show the raw json but also have instructions for the builder card!

Reviewer 3: 👍

{

>   <INSERT SAMPLE CONFIGURATION(S)>

Reviewer 2: ```suggestion
  TODO: INSERT SAMPLE CONFIGURATION(S)
```

}
```

### Next Steps

> Add any additional information you want them to know and direct them towards what to do next with Viam.

Reviewer 2: This is a template that anyone should be able to use, we can't expect the general public to think about what other people should do next with Viam. 
```suggestion
_Add any additional information you want readers to know and direct them towards what to do next with this module._
```

> For example: 
> 
> - To test your...
> - To write code against your...

Reviewer 2: ```suggestion
_For example:_ 

- To test your...
- To write code against your...
```

## Troubleshooting

> Add troubleshooting notes here.

Reviewer 2: ```suggestion
_Add troubleshooting notes here._
```
